spec/models/redeemer_spec.rb

```diff
@@ -10,6 +10,10 @@
 
     before :each do
       allow(Current).to receive(:user).and_return(user)
+      # Stub the specific helper method to prevent it from accessing the filesystem.
+      # Since image tags are not relevant for our tests, we replace
+      # them by empty strings
+      allow_any_instance_of(EmailHelper).to receive(:email_image_tag).and_return("")
```

Comment:
This fix is placed inside the `redeemer_spec.rb`, could other test files profit from it as well?

In my opinion, retrieving these assets should work in tests as well and I'd like to include that in the tests. If this doesn't work, then something isn't set up correctly for the Vite configuration. (In CI/CD, assets get precompiled, see the env variable `IN_CI_CD`).

Reply:
This shouldn't be a problem anymore with the new Vite & Dev Container setup. If it still is, I'd like to tackle that in a new PR.